aiida/README.md

```diff
@@ -0,0 +1,38 @@
+# {{ name }}
+{% if description %}
+{{ description }}
+{% endif %}
+
+{% if archive_url %}
+This project imports an [AiiDA archive]({{ archive_url }}) upon starting an environment, which may take some time.
+{% endif %}
+
+
+### Introduction
+
+This project comes with an AiiDA environment already pre-configured for your user.
+
+ 1. Start a new environment (e.g. 1 CPU, 2GB memory)
+ 1. (optional) inspect the `aiida_config.yaml` in case you would like to make any changes
+ 1. Open the terminal and type
+    ```
+    verdi status
+    ```
+
+You should see a status similar to
+```
+ ✔ config dir:  /work/{{ name }}/repo/.aiida
```

Comment:
you should replace `name` with `__sanitized_project_name__` wherever the directory name of the project is needed. If you create a project with spaces (or other special characters) in it, those will be a part of `name` but `__sanitized_project_name__` will have them replaced - this is the same as will be used for the project slug and URL. See e.g. https://github.com/rokroskar/contributed-project-templates/blob/add-aiida/aiida/README.md

Reply:
thanks, fixed!